engine/Default/smr.inc

```diff
@@ -23,6 +23,17 @@ function smrBBCode($bbParser, $action, $tagName, $default, $tagParams, $tagConte
 	global $overrideGameID, $disableBBLinks, $player, $account, $var;
 	try {
 		switch($tagName) {
+			case 'combatlog':
+				$logID = (is_numeric($default)) ? $default : $tagParams['id'];
+				if ($action == BBCODE_CHECK) {
+					return true;
+				}
+				$container = create_container('skeleton.php', 'combat_log_viewer.php');
+				$container['action'] = 5;
```

Comment:
I'd love to see these as named constants because the numbers are not the easiest to associate with actions (although I know 5 is 'view')